docs/src/operator/usage.md

```diff
@@ -59,27 +59,17 @@ Create a faucet account for the faucet app to use - or skip this step if you alr
 ```sh
 mkdir accounts
 miden-faucet create-faucet-account \
-  --token-symbol MY_TOKEN \
+  --token-symbol BTC \
   --decimals 12 \
   --max-supply 5000
```

Comment:
nit: if we are using BTC as the example, should we use other values consistent with it? - i.e., $6$ for decimal places and $2100000000000000$ for `max-supply`